allensdk/brain_observatory/behavior/data_objects/stimuli/templates.py

```diff
@@ -30,13 +31,21 @@
 
 class Templates(DataObject, StimulusFileReadableInterface,
                 NwbReadableInterface, NwbWritableInterface):
-    def __init__(self, templates: StimulusTemplate):
+    def __init__(self, templates: Dict[str, StimulusTemplate]):
         super().__init__(name='stimulus_templates', value=templates)
+        self.image_template_key = None
+        self.movie_template_key = None
+        for key in templates.keys():
```

Comment:
this bit of code does not generalize if there were to be added an additional movie. Please name `movie_template_key` specifically as `fingerprint_movie_template_key`. I believe you should also prepend the fingerprint key with `fingerprint` to distinguish it from another movie.